app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,3 @@
+class IngredientsController < ApplicationController
+  def show; end
```

Comment:
For all the ingredients that a chef uses, you could use an `index` here, or choose to create a nested controller, where you have a `chefs` folder and inside you have an `ingredients_controller` with an index action. That design helps indicate that the list of ingredients in the chef/ingredients_controller are particular to one specific chef